test/index.js

```diff
@@ -353,6 +381,24 @@ describe('KeyringController', function () {
       // was also removed after removing the account
       expect(keyringController.keyrings).toHaveLength(1);
     });
+
+    it('does not remove the keyring if there are accounts remaining after removing one from the keyring', async function () {
+      // Add a new keyring with one account
```

Comment:
This comment says that we're adding a keyring with 1 account, but the test is asking the keyring controller to generate 2 accounts.

I'm guessing that the test is correct but the comment is wrong:

```suggestion
      // Add a new keyring with two accounts
```

Reply:
done [here](https://github.com/MetaMask/KeyringController/pull/140/commits/3b35dceaf4c0a1170f516d19b41d7f5f8bd57c93)

Reply:
whoops: make `account` plural [here](https://github.com/MetaMask/KeyringController/pull/140/commits/f740476e8472d62acda0b28551312e8d04504f0a)